pyairctrl/airctrl.py

```diff
@@ -216,6 +253,8 @@ def get_wifi(self):
         pprint.pprint(wifi)
 
     def get_firmware(self):
+        # url = 'http://{}/di/v1/products/1/device'.format(self._host)
+        # url = 'http://{}/di/v1/products/1/userinfo'.format(self._host)
```

Comment:
why putting these comments?

Reply:
:-)
These URL where sniffed during testing.
I will add support for the first one in next PR...

Second URL is not working on my device (?)

Reply:
Sure, patches are welcome

Reply:
removed